src/CodeGenerator/Program.cs

```diff
@@ -8,37 +8,83 @@
 using System.Text;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
+using System.CommandLine;
 
 namespace CodeGenerator
 {
     internal static class Program
     {
-        static void Main(string[] args)
+        private const string InternalNamespace = ".Internal";
+
+        static async Task<int> Main(string[] args)
         {
-            string outputPath;
```

Comment:
I'm unsure why github has decided to diff this in this way, but this is all new code apart from the declarations of the `outputPath` and `libraryName` variables which was moved up from below. You can see this code in raw format in this comment. This does a few things:

01. Creates three parameters
01. Adds them to the command line parser
01. Runs the parser
01. If the parser passes, it moves on to the normal logic you had in this file which now starts on line 89

https://github.com/mellinoe/ImGui.NET/blob/70a7b35459b260377b7487981b65839a02e3a8c6/src/CodeGenerator/Program.cs#L21-L87